airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/paginators/strategies/page_increment.py

```diff
@@ -45,4 +53,10 @@ def reset(self):
         self._page = self.start_from_page
 
     def get_page_size(self) -> Optional[int]:
-        return self.page_size
+        if self._page_size:
+            page_size = self._page_size.eval(self.config)
```

Comment:
can you add two tests to https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/unit_tests/sources/declarative/requesters/paginators/test_page_increment.py#L23
1. with a page size evaluated from the config
2. with a page size that does not evaluate to an integer to show the error handling

Reply:
Updated